certbot-apache/certbot_apache/constants.py

```diff
@@ -131,6 +131,16 @@
 MOD_SSL_CONF_DEST = "options-ssl-apache.conf"
 """Name of the mod_ssl config file as saved in `IConfig.config_dir`."""
 
+
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-options-ssl-apache-conf-digest.txt"
+"""Name of the hash of the updated or informed mod_ssl_conf as saved in `IConfig.config_dir`."""
+
+ALL_SSL_OPTIONS_HASHES = [
+    '2086bca02db48daf93468332543c60ac6acdb6f0b58c7bfdf578a5d47092f82a',
+    '4844d36c9a0f587172d9fa10f4f1c9518e3bcfa1947379f155e16a70a728c21a',
```

Comment:
Oh wow. Thanks for remembering to get the hash of that file. Given I forgot about that, I'm beginning to see why the Apache plugin is much more popular on Debian based systems...

The other hashes are from before the `letsencrypt -> certbot` rename. In general, I ran `git log <file>`, got the hash of the file from every commit, on the rename commit, ran `git checkout HEAD~1`, then went to `letsencrypt-apache/letsencrypt_apache` and did the same thing.

I've updated my PR to include the hash of `centos-options-ssl-apache.conf`. It doesn't appear like it's ever been modified before.

Two things I just realized though:

1. We should add comments to the top of the Apache ssl-options files like we did for Nginx.
2. The comment in Nginx is out of date because we no longer create the `.new` file.

Let me know if you'd like me to take either piece of that.